lua/telescope/builtin/init.lua

```diff
@@ -402,6 +402,8 @@ local apply_config = function(mod)
         end
       end
 
+      opts.current_filetype = vim.bo.filetype
```

Comment:
is there a better way to do this?

Reply:
No its the correct way to do this, but why here? why at this place and not the beginning of `live_grep` this will now apply to all pickers and only `live_grep` needs it

Reply:
I'll try again. I *think* the first time I used it, I did have it in there and it came out blank. I guessed it must be b/c the floating window is up and the buffer is now telescope. I was using vm.fn.expand("%:e") to get it though, maybe that was not the right way? I'll try and get back to you.

Reply:
Nevermind. Got it now thanks to your other comment.